presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoPreparedStatement.java

```diff
@@ -849,4 +856,28 @@ private static String typedNull(String prestoType)
     {
         return format("CAST(NULL AS %s)", prestoType);
     }
+
+    private static List<ColumnInfo> getDescribeOutputColumnInfoList(ResultSet resultSet)
+            throws SQLException
+    {
+        ImmutableList.Builder<ColumnInfo> list = ImmutableList.builder();
+        while (resultSet.next()) {
+            String columnName = resultSet.getString("Column Name");
+            String catalog = resultSet.getString("Catalog");
+            String schema = resultSet.getString("Schema");
+            String table = resultSet.getString("Table");
+            ClientTypeSignature clientTypeSignature = new ClientTypeSignature(resultSet.getString("Type"));
+            ColumnInfo.Builder builder = new ColumnInfo.Builder()
+                    .setColumnName(columnName)
+                    .setColumnLabel("")
```

Comment:
can we set the `columnLabel` as `columnName` ?

Reply:
@Praveen2112 yea we can do that, but I wonder what's the difference between column name & label? Also, do you think we should have `Type Size` and `Aliased` in the `columnInfo`, these two fields are missing however it is shown in the output of [DESCRIBE OUTPUT](https://prestodb.github.io/docs/current/sql/describe-output.html).

Reply:
The column label is used for aliased column name and it will be the same as column name if omitted. Current ResultSetMetaData#getColumnLabel from `Statement` (≒PreparedStatement) behave as @Praveen2112 suggestion, so I agree with `.setColumnLabel(columnName)`. 
https://docs.oracle.com/javase/8/docs/api/java/sql/ResultSetMetaData.html#getColumnLabel-int-

In my opinion, It would be better to add `setColumnDisplaySize` here, but it seems varchar returns 0.

Reply:
@ebyhr Thank you for your advice, I think `setColumnDisplaySize` is not used for setting the `Type Size`. For example, for a boolean type, its type size is 1, but its display size is 5.